src/components/RunMap/index.jsx

```diff
@@ -16,6 +16,10 @@ import RunMarker from './RunMaker';
 import RunMapButtons from './RunMapButtons';
 import styles from './style.module.scss';
 
+//change 'dispaly' to 'hide' if you want to hide the road label;
+const RoadLabelDisplay = 'display';
```

Comment:
make it to `bool`

Reply:
changed to 'true' and 'false'

Reply:
not that way, you are still using string.